fee_allocator/accounting/chains.py

```diff
@@ -489,8 +506,17 @@ def alliance_noncore_fees_collected(self) -> Decimal:
 
     @property
     def alliance_noncore_to_dao_usd(self) -> Decimal:
-        return self.alliance_noncore_fees_collected * self.chains.alliance_config.alliance_fee_allocations["non_core"].dao_share_pct
+        beets_share_pct = self.chains.beets_share_pct if self.name == "optimism" else 0
+        return self.alliance_noncore_fees_collected * self.chains.alliance_config.alliance_fee_allocations["non_core"].dao_share_pct  * (1 - beets_share_pct)
 
     @property
     def alliance_noncore_to_vebal_usd(self) -> Decimal:
-        return self.alliance_noncore_fees_collected * self.chains.alliance_config.alliance_fee_allocations["non_core"].vebal_share_pct
+        beets_share_pct = self.chains.beets_share_pct if self.name == "optimism" else 0
+        return self.alliance_noncore_fees_collected * self.chains.alliance_config.alliance_fee_allocations["non_core"].vebal_share_pct  * (1 - beets_share_pct)
+
+    @property
+    def alliance_noncore_to_beets_usd(self) -> Decimal:
+        beets_share_pct = self.chains.beets_share_pct if self.name == "optimism" else 0
+        if beets_share_pct == 0:
+            return Decimal(0)
+        return self.alliance_noncore_fees_collected * beets_share_pct * (1 - self.chains.alliance_config.alliance_fee_allocations["non_core"].partner_share_pct)
```

Comment:
why `(1 - self.chains.alliance_config.alliance_fee_allocations["non_core"].partner_share_pct)` here? @jalbrekt85 can you double check?